[26/Jan/2020 12:39:30] ERROR [django.request:228] Internal Server Error: /challenge/5197626439671097/
Traceback (most recent call last):
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/smartif.py", line 175, in translate_token
    op = OPERATORS[token]
KeyError: 'challenge.statement_10q|filename'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/views/generic/base.py", line 97, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/django/trainer/website/investmenttrainer/challenge/views.py", line 18, in get
    return render(request, self.template_name, {"challenge": challenge, 'sections': sections, "challenge_type": challenge_type})
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/loaders/cached.py", line 53, in get_template
    template = super().get_template(template_name, skip)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 156, in __init__
    self.nodelist = self.compile_nodelist()
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 194, in compile_nodelist
    return parser.parse()
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 477, in parse
    raise self.error(token, e)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 475, in parse
    compiled_result = compile_func(self, token)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/loader_tags.py", line 266, in do_extends
    nodelist = parser.parse()
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 477, in parse
    raise self.error(token, e)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 475, in parse
    compiled_result = compile_func(self, token)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/loader_tags.py", line 209, in do_block
    nodelist = parser.parse(('endblock',))
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 477, in parse
    raise self.error(token, e)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 475, in parse
    compiled_result = compile_func(self, token)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/defaulttags.py", line 811, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 477, in parse
    raise self.error(token, e)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 475, in parse
    compiled_result = compile_func(self, token)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/defaulttags.py", line 960, in do_if
    nodelist = parser.parse(('elif', 'else', 'endif'))
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 477, in parse
    raise self.error(token, e)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 475, in parse
    compiled_result = compile_func(self, token)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/defaulttags.py", line 951, in do_if
    condition = TemplateIfParser(parser, bits).parse()
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/defaulttags.py", line 884, in __init__
    super().__init__(*args, **kwargs)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/smartif.py", line 166, in __init__
    mapped_tokens.append(self.translate_token(token))
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/smartif.py", line 177, in translate_token
    return self.create_var(token)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/defaulttags.py", line 887, in create_var
    return TemplateLiteral(self.template_parser.compile_filter(value), value)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 562, in compile_filter
    return FilterExpression(token, self)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 656, in __init__
    filter_func = parser.find_filter(filter_name)
  File "/root/.virtualenvs/venv/lib/python3.6/site-packages/django/template/base.py", line 568, in find_filter
    raise TemplateSyntaxError("Invalid filter: '%s'" % filter_name)
django.template.exceptions.TemplateSyntaxError: Invalid filter: 'filename'
